### PR TITLE
Config with pure python

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -75,22 +57,27 @@ def alltests():
...
     package_dir={'': 'src'},
     namespace_packages=['zope'],
+    python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*',
     install_requires=[
         'setuptools',
         'zope.interface',
     ],
     tests_require=[
         'zope.testrunner',
     ],
```
Comment (mgedmin) [inline setup.py:76]: Do we still want this?

(It's ironic that we had tests_require as a global, used it for extras_require['test'], and then forgot to use it for tests_require.)

**Review by mgedmin — APPROVED** [review]
LGTM

File: setup.py (hunk 1 above)
Comment (icemac) [reply setup.py:76]: I was about to remove it but than seem to have forgotten to actually do it. As it is no longer needed I'll remove it.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**